client/src/components/Button/SubmitButton.js

```diff
@@ -1,54 +1,26 @@
-import styled, { css } from "styled-components";
-import BaseButton from "./BaseButton";
-import { theme } from "constants/theme";
-const { button } = theme;
-
-// props: inline, primary, primarylight, secondary, tertiary
-
-const SubmitButton = styled(BaseButton).attrs(({ size, inline }) => {
-  return { size, inline };
-})`
-  ${button.regular}
```

Comment:
Why are these removed?

Reply:
@snowiesuet @alexjmcm There have been changes to this file prior to this PR. Please always pull and rebase with master and work with the latest changes.